aixplain/modules/pipeline_response.py

```diff
@@ -0,0 +1,54 @@
+from dataclasses import dataclass
+from typing import Any, Optional, Dict, List, Union, Text
+from aixplain.enums import Status 
+
+@dataclass
+class PipelineResponse:
+    
+    def __init__(
+        self,
+        status: Status, 
+        error: Optional[Dict[str, Any]] = None,
+        elapsed_time: Optional[float] = 0.0,
+        data: Optional[Text] = None,
+        url: Optional[Text]="",
+        **kwargs,  
+    ):
+        self.status = status
+        self.error = error
+        self.elapsed_time = elapsed_time
+        self.data = data 
+        self.additional_fields = kwargs
+        self.url=url
+    def __getitem__(self, key: str) -> Any:
```

Comment:
using a auto formatter (eg Black) would help to avoid for such cases. For example a spacing is needed here to make the code more readable.

Reply:
Well we don't want to break any code that's already considering the response as a dictionary right? It's safer to keep both options I think.

Reply:
if it's the case, that's ok